vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/openid/PresentationFactory.kt

```diff
@@ -165,6 +204,31 @@ internal class PresentationFactory(
         )
     }
 
+    private fun calcSessionTranscript(
+        dcApiRequest: Oid4vpDCAPIRequest,
+        nonce: String,
+        jsonWebKeys: Collection<JsonWebKey>?,
+        responseWillBeEncrypted: Boolean
+    ): SessionTranscript {
+        //if (responseWillBeEncrypted != dcApiRequest.) TODO check if response mode is dcapi-jwt
+        val jwkThumbprint = if (responseWillBeEncrypted && !jsonWebKeys.isNullOrEmpty()) {
+            //TODO what if there are multiple JSON web keys?
```

Comment:
Select the first one that is an EC key, or has key use `enc` or something like that?